ci/index.md

```diff
@@ -5,3 +5,4 @@ title: How to implement continuous integration for redhat-cop repos
 This section talks through various continuous integration strategies we employ across the repositories in this organization.
 
 * [Publishing Container Images to Quay.io](./publishing-images.html)
+* [Resource Management](./resource-management.md)
```

Comment:
Can we come up with something more descriptive than "resource management"? Something like "requesting repository provisioning and access". Otherwise I don't think many people will click on it.

Reply:
@etsauer updated